app/common/elasticsearchclient.py

```diff
@@ -2874,24 +3030,38 @@ class AggregationBuilder(object):
                              FilterTypes.SCORE_RANGE,
                              ]
 
+    _EVIDENCE_FACET_MAP = {
```

Comment:
If we create a facet, we should also create a corresponding filter, otherwise it is not usable from the UI. For example you can see there are 10 documents with a paper published in 2016, but you can't select them with a query.